src/status_im/native_module/module.cljs

```diff
@@ -19,5 +19,6 @@
   (-clear-web-data [this])
   (-call-web3 [this host payload callback])
   (-module-initialized! [this])
-  (-should-move-to-internal-storage? [this callback]))
+  (-should-move-to-internal-storage? [this callback])
+  (-notify [this token callback]))
```

Comment:
shouldn't we call status-go here?

Reply:
Isn't this just a collection of functions with their signatures? I don't see any other calls doing stuff here.